CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.0-alpha.0 - 2025-02-27
```

Comment:
This is extremely close to release IMO. I wouldn't be even annoyed if there was push to skip the pre-release. (To be clear, yes, let's do a pre-release just to be absolutely sure.) So I think `rc` would be more accurate name - this definitely is a release candidate in my mind.

But it's not super important to me, just something to reduce confusion.